src/Log/SentryLogger.php

```diff
@@ -332,4 +333,80 @@ public static function backtrace(array $record) : array
         ]);
     }
 
+    /**
+     * Get the information about the current release, if possible
+     * @return array
+     */
+    protected function getGitInfo()
+    {
+        // Initialise empty array to return
+        $return = [];
+        // If the file operations error out, we need to catch it
+        try {
+            $return = $this->getGitCommitMessage($return);
+        } catch (\Exception $exception) {
+            // Default message. As it's an extra, it's not overly crowding the interface
+            $return['Message'] = 'No git repo found or inaccessible';
+        }
+
+        return $return;
+    }
+
+    /**
+     * Get the HEAD tag or the release
+     *
+     * @return string
+     */
+    public static function getGitHead()
+    {
+        $head = file_exists(Director::baseFolder() . '/.git/HEAD');
```

Comment:
I pulled your branch in order to test this, but the logic there looks different (Misplaced `/`) resulting in file not found. Not sure what that's about.